smart-contract/src/contract.rs

```diff
@@ -656,33 +240,37 @@ fn try_transfer(
     // Ensure the sender holds at least the indicated amount of dcc.
     let balance = deps
         .querier
-        .query_balance(info.sender.clone(), &state.dcc_denom)?;
+        .query_balance(info.sender.clone(), &state.denom)?;
     if balance.amount < amount {
         return Err(contract_err("insufficient dcc balance in transfer"));
```

Comment:
can we change these error messages to not use `dcc` - it might not be clear to omnibus usdf users what that means.

Reply:
any recommended terminology? inject the state denom?

Reply:
I actually it would be fine to say insufficient balance to transfer or insufficient balance to burn because we're just using the one denom now.